source/common/upstream/BUILD

```diff
@@ -426,6 +426,8 @@ envoy_cc_library(
         "//source/common/protobuf:utility_lib",
         "//source/common/runtime:runtime_lib",
         "//source/server:transport_socket_config_lib",
+        "//source/extensions/load_balancing_policies/ring_hash:ring_hash_lb_lib",
```

Comment:
We moved these out to extensions so we didn't need the hard link.  I'd prefer we avoid adding them back in.

Reply:
got it, I can split it out